src/service/video-manager-impl.js

```diff
@@ -186,13 +191,6 @@ export class VideoManager {
    * @private
    */
   maybeInstallVisibilityObserver_(entry) {
-    // TODO(aghassemi): Remove this later. For now, the visibility observer
-    // only matters for autoplay videos so no point in monitoring arbitrary
-    // videos yet.
-    if (!entry.hasAutoplay) {
```

Comment:
Removed in this PR in order to allow all videos to trigger the `video-session-visible` trigger.